extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -623,7 +624,7 @@ int ConsumeWindowsEventLog::processQueue(const std::shared_ptr<core::ProcessSess
 
   auto before_time = std::chrono::high_resolution_clock::now();
   utils::ScopeGuard timeGuard([&](){
-    logger_->log_debug("processQueue processed %d Events in %llu ms",
+    logger_->log_debug("processQueue processed %d Events in %" PRIu64 " ms",
```

Comment:
chrono durations are sadly not a good fit with printf-style format strings. The standard doesn't specify the return type of `std::chrono::milliseconds::count`, only that it's a signed integer type of at least 45 bits.

I recommend wrapping the return value in `int64_t{ ... }` and using `PRId64` as a format specifier. With list-initialization, we get a compiler error if a narrowing conversion would be necessary to convert the return value to `int64_t` and until then we can assume that we're safe and don't lose any data.

This applies to both occurences of printing durations.

Reply:
There was a similar issue in #741: https://github.com/apache/nifi-minifi-cpp/pull/741#discussion_r386539946

Reply:
fixed in 61107c8b